app/build.gradle.kts

```diff
@@ -7,15 +7,19 @@ plugins {
     id(Config.ApplyPlugins.KSP)
     id(Config.ApplyPlugins.PARCELIZE)
 }
+// module specific code coverage verification threshold
+extra.set("jacocoCoverageThreshold", 0.40.toBigDecimal())
 
-extra.set("jacocoCoverageThreshold", 0.40.toBigDecimal()) // module specific code coverage verification threshold
-
-apply(from = "../renameAppBundle.gradle.kts") // configures additional gradle tasks to rename app bundles (when needed)
+apply(from = "../renameAppBundle.gradle.kts") // configures additional gradle tasks to rename
+// app bundles (when needed)
```

Comment:
@Josh-Gebbeken nit: It looks like some file-wide code cleanup caused much of this file to be modified (likely due to a lower characters per line default). I'd say undo all of the changes to this file made in this commit and just redo the actual change of applicationId to `com.bottlerocketstudios.mapsdemo` in the release buildtype section as some of them are a little excessive (going from 2 lines for comment and code for the keystore file to 9 lines)

Reply:
fixed

Reply:
@Josh-Gebbeken After your commits mentioned that ktlint performed these formatting changes, it reminded me of the .editorconfig file which manages max line length. I'd recommend you bring in [the file from the architecture demo](https://github.com/BottleRocketStudios/Android-ArchitectureDemo/blob/develop/.editorconfig) and re-run ktlint (it should respect the line length value of 200 and not auto format this file with an 80 character length limit) :)

Reply:
File added to project and ktlint formatted and ran with no issues.